### PR TITLE
Created the GameManagerTester

### PR DESCRIPTION
**Pull request by BeckDun**

### CODE REVIEW

File: src/main/java/GameManagerTester.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import java.awt.event.KeyEvent;
+import java.util.Collections;
+import java.util.Scanner;
+
+public class GameManagerTester {
+    public static void main(String[] args) {
+        System.out.println("This is the GameManager tester class!");
+        
+        GameManager2 game = new GameManager2();
+        game.startGame(4);
```
Comment (dmdunla) [inline src/main/java/GameManagerTester.java:13]: Add a `Scanner` here to get this input from user

File: src/main/java/GameManagerTester.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        // System.out.println("Game sequence: " + game.getSequence());
```
Comment (dmdunla) [inline src/main/java/GameManagerTester.java:14]: Remove comments

File: src/main/java/GameManagerTester.java [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        while(!game.isGameOver()) {
```
Comment (dmdunla) [inline src/main/java/GameManagerTester.java:15]: Before the main loop, print the list of valid keys for users to choose from

File: src/main/java/GameManagerTester.java [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+import java.util.ArrayList;
+import java.awt.event.KeyEvent;
+import java.util.Collections;
+import java.util.Scanner;
+
+public class GameManagerTester {
+    public static void main(String[] args) {
+        System.out.println("This is the GameManager tester class!");
+        
+        GameManager2 game = new GameManager2();
```
Comment (dmdunla) [inline src/main/java/GameManagerTester.java:12]: Switch to GameManager once you have migrated the GameManager2 code to there.

**Review by dmdunla — CHANGES_REQUESTED** [review]

File: src/main/java/GameManagerTester.java (hunk 4 above)
Comment (BeckDun) [reply src/main/java/GameManagerTester.java]: Done

File: src/main/java/GameManagerTester.java (hunk 2 above)
Comment (BeckDun) [reply src/main/java/GameManagerTester.java:14]: Done

File: src/main/java/GameManagerTester.java (hunk 1 above)
Comment (BeckDun) [reply src/main/java/GameManagerTester.java]: done. commited in d2837113442da553042878c2e94758f401621b2d

File: src/main/java/GameManagerTester.java (hunk 3 above)
Comment (BeckDun) [reply src/main/java/GameManagerTester.java:20]: done. see commit in 6e4ec36033d296bf44c8aa9a355ab4e0d10ece1e

**Review by dmdunla — APPROVED** [review]

**Pull request merged**